offload/liboffload/include/generated/OffloadEntryPoints.inc

```diff
@@ -439,3 +439,916 @@ ol_result_t olGetDeviceInfoSizeWithCodeLoc(ol_device_handle_t Device,
   currentCodeLocation() = nullptr;
   return Result;
 }
+
+///////////////////////////////////////////////////////////////////////////////
+ol_impl_result_t olMemAlloc_val(ol_device_handle_t Device, ol_alloc_type_t Type,
+                                size_t Size, void **AllocationOut) {
+  if (true /*enableParameterValidation*/) {
```

Comment:
Question: Is this a flag set when generating the files or picked up from some template?
Wonder if we want a switch to turn this off.

Reply:
This is now controlled with the `OFFLOAD_DISABLE_VALIDATION` env var. It defaults to on, and the unit tests expect it to be on.